station_data.py

```diff
@@ -62,45 +63,52 @@ def get_events(station, start, end):
 
         # Do NOT convert date and time to Python formats as they are not JSON
         # serializable
-        event['date'] = row[0]
-        event['time'] = row[1]
 
-        event['timestamp'] = int(row[2])
-        event['nanoseconds'] = int(row[3])
+        event['location'] = {
+                             'latitude'  : station_data['latitude'],
+                             'longitude' : station_data['longitude'],
+                             'altitude'  : station_data['altitude']
+                            }
+
+        event['time'] = row[0] + ' ' + row[1] + '%9d'%(int(row[2]))
```

Comment:
I would add a dot between the number of seconds and nanoseconds. (to indicate the nanoseconds are fractional seconds)